src/httprpc.cpp

```diff
@@ -220,7 +220,7 @@ static bool InitRPCAuthentication()
         LogPrintf("No rpcpassword set - using random cookie authentication\n");
         if (!GenerateAuthCookie(&strRPCUserColonPass)) {
             uiInterface.ThreadSafeMessageBox(
-                _("Error: A fatal internal error occurred, see debug.log for details"), // Same message as AbortNode
+                _("A fatal internal error occurred, see debug.log for details: ") + "Unable to open cookie authentication file for writing", // Same message as AbortNode
```

Comment:
Should the cookie message be translated too? I understand why this message is split into parts.